tests/base/eval_model_valid_epoch_ends.py

```diff
@@ -34,8 +34,9 @@ def validation_epoch_end(self, outputs):
 
             val_acc_mean += val_acc
 
-        val_loss_mean /= len(outputs)
-        val_acc_mean /= len(outputs)
+        if outputs:  # skip zero divisions
```

Comment:
Does this cover len(outputs) == 0 as well?

Reply:
yes :]
```
output = []
bool(output)
Out[3]: False
```